src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/VirtualMachinesOverviewTabDetailsTitle.tsx

```diff
@@ -46,11 +46,18 @@ const VirtualMachinesOverviewTabDetailsTitle: React.FC<
               <CopyIcon />
             </span>
           </DropdownItem>,
-          <DropdownItem key="stop" onClick={() => (isMachineStopped ? startVM(vm) : stopVM(vm))}>
-            {isMachineStopped ? t('Resume VirtualMachine') : t('Stop VirtualMachine')}
+          <DropdownItem
+            isDisabled={!(isMachineRunning || isMachinePaused)}
+            key="stop"
+            onClick={() => stopVM(vm)}
+          >
+            {t('Stop VirtualMachine')}
+          </DropdownItem>,
+          <DropdownItem isDisabled={!isMachinePaused} key="resume" onClick={() => unpauseVM(vm)}>
+            {t('Resume VirtualMachine')}
           </DropdownItem>,
-          <DropdownItem key="pause" onClick={() => (isMachinePaused ? unpauseVM(vm) : pauseVM(vm))}>
-            {isMachinePaused ? t('Unpause VirtualMachine') : t('Pause VirtualMachine')}
+          <DropdownItem isDisabled={!isMachineRunning} key="pause" onClick={() => pauseVM(vm)}>
+            {t('Pause VirtualMachine')}
```

Comment:
the start VM is removed? why?
That's one of the most common actions (if not the most common of them all) that can be done in a VM

Reply:
If you look closely, you will see that the "Start VirtualMachine" button was not there at all even before this code change. You think start VM was removed because I deleted `startVM` call, which was used for resuming VM - and that is what I consider not appropriate, because there is also `unpauseVM` - I consider that one more accurate.

Reply:
I'm not sure why it was named `Resume` and I honestly don't think that's important.
I think there should be 3 options as there were before. (the SSH copy command should stay always)
If VM is stopped - you should show Start VM (rename the resume to start), and Pause VM (disabled).
If VM is running - you should show Stop VM and Pause VM both active
if VM is paused - you can either Unpause or Stop the VM.
This will also be consistent with current VM actions dropdown

Reply:
Maybe there was some reason to use Resume, originally? I think we'd need some design and discussion with involved people, to make sure we are fixing this the correct way...

Reply:
i think i initially implemented this feature, you can always look on git blame :) if I remember correctly I thought it is strange it was a resume and when asked about it they told me that what they currently want. i dont think there is really a big issue to change it to something more appropriate